exercises/part3/semantic-parsing-seq2seq/predictor_setup.py

```diff
@@ -0,0 +1,14 @@
+from typing import List
+from allennlp.predictors import Predictor
+from allennlp.models.archival import load_archive
+from allennlp_models.generation import ComposedSeq2Seq  # Need this for loading model archive
+
+from nla_semparse.nla_semparse.nla_metric import NlaMetric  # Need this for loading model archive
+
+
+ARCHIVE = load_archive("nla_semparse/trained_models/seq2seq_model.tar.gz")
+PREDICTOR = Predictor.from_archive(ARCHIVE, "seq2seq")
+
+def translate_nla(source: str) -> List[str]:
+    prediction_data = PREDICTOR.predict_json({"source": source})
+    return prediction_data["predicted_tokens"]
```

Comment:
I'd recommend moving these lines into the source blocks, even though they are shared.  It makes the source blocks easier to follow.  The setup is intended for things that you shouldn't have to think about or understand in order to follow what's in the source block.  That's not true of these things.

Also, I think you're using all caps because this are "global" variables?  But this is a tiny demonstration script, and those rules don't apply.  The all caps just look really out of place here.